config/initializers/open-weather-api.rb

```diff
@@ -0,0 +1,2 @@
+# API key for weather
+# config.api_key = "6157d0b45dbc65d57d7bcb4569934b10"
```

Comment:
Delete the comment so that nobody can see your API key.

Reply:
Where is your key stored now? 
You should do the same set up as for cloudinary, to not add it on github.